src/sql/parts/dashboard/pages/dashboardPageContribution.ts

```diff
@@ -6,11 +6,20 @@
 import { Registry } from 'vs/platform/registry/common/platform';
 import { IJSONSchema } from 'vs/base/common/jsonSchema';
 import { Extensions, IDashboardWidgetRegistry } from 'sql/platform/dashboard/common/widgetRegistry';
+import { mixin } from 'vs/base/common/objects';
 
 let widgetRegistry = <IDashboardWidgetRegistry>Registry.as(Extensions.DashboardWidgetContribution);
 
+export function GenerateDashboardWidgetSchema(type?: 'database' | 'server', extension?: boolean): IJSONSchema {
```

Comment:
should this be named `generateDashboardWidgetSchema` per project coding conventions?